bonding-rose.lic

```diff
@@ -0,0 +1,114 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#bonding-rose
+=end
+
+custom_require.call(%w[common common-items common-money common-travel drinfomon equipmanager])
+
+class BondingRose
+  include DRC
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    @settings = get_settings
+
+    unless DRStats.cleric?
+      echo '***MUST BE A CLERIC***'
+      exit
+    end
+
+    make_rose
+  end
+
+  def make_rose
+    temp_settings = @settings
+    temp_settings.hometown = 'Crossing'
+    ensure_copper_on_hand(1000, temp_settings)
```

Comment:
Is 1000 really enough for potentially buying six items?

Reply:
yeah, it's about 880 kronars total buying everything but I made it 1000 just to be sure you have enough for the 2 ferry trips involved also.

Reply:
Sounds good